File: fontes/cms/src/models/banner/banner-update.model.ts
export class BannerUpdateModel {
    public constructor(init?: Partial<BannerUpdateModel>) {
        Object.assign(this, init);
    }

    id: number;
    nomeImagem: string;
    nomeImagemMobile: string;
    titulo: string;
    subtitulo: string;
    area: string;
    tempoExibicao: number;
    descricao: string;
    rota: string;
    linkExterno: string;
    arquivo: File;
    arquivoMobile: File;
    nomeLink: string;
    caminhoDesktop: string;
    caminhoMobile: string;
    ativo: string;

}
export class BannerListModel {
    constructor(init?: Partial<BannerListModel>) {
        Object.assign(this, init);
    }

    count: number;
    result: BannerListModel[];
}

File: fontes/api/CarePlusAPI/CarePlusAPI/Models/Banner/BannerUpdateModel.cs
using Microsoft.AspNetCore.Http;
using System.ComponentModel.DataAnnotations;
using System.Diagnostics.CodeAnalysis;

namespace CarePlusAPI.Models.Banner
{
    [ExcludeFromCodeCoverage]
    public class BannerUpdateModel
    {
        [Required]
        public int? Id { get; set; }
        [StringLength(100)]
        [Required]
        public string Titulo { get; set; }
        [StringLength(100)]
        public string Subtitulo { get; set; }
        [StringLength(150)]
        [Required]
        public string Area { get; set; }
        public int TempoExibicao { get; set; }
        [StringLength(255)]
        public string Descricao { get; set; }
        [Required]
        [StringLength(100)]
        public string Rota { get; set; }
        [Required]
        public char LinkExterno { get; set; }
        [StringLength(255)]
        public string Link { get; set; }
        [Required]
        public char Ativo { get; set; }
        public IFormFile Arquivo { get; set; }
        public IFormFile ArquivoMobile { get; set; }
        [StringLength(255)]
        public string CaminhoDesktop { get; set; }
        [StringLength(255)]
        public string NomeImagemDesktop { get; set; }
        [StringLength(255)]
        public string CaminhoMobile { get; set; }
        [StringLength(255)]
        public string NomeImagemMobile { get; set; }
        public int Ordem { get; set; }

    }
}